tests/helper/helper_oc.go

```diff
@@ -577,7 +577,7 @@ func (oc OcRunner) GetNamespaceProject() string {
 }
 
 func (oc OcRunner) HasNamespaceProject(name string) bool {
```

Comment:
You are changing the function `HasNamespaceProject` to use the List operation instead of Get, it will have an effect for all the tests calling this function. But we see that the change is not transparent, and could introduce other flaky tests.

It would be safer to create another function using the List operation, for example a `ListNamespacesProjects` that would return the list of namespaces/projects. And you could check for the existence of requested name on the value returned by the function.